pkg/cleaner/jobcontroller.go

```diff
@@ -75,10 +74,19 @@ func NewCleanupJob(bd *v1alpha1.BlockDevice, volMode VolumeMode, tolerations []v
 	mountName := "vol-mount"
 
 	if volMode == VolumeModeBlock {
+		jobContainer.Command = []string{"/bin/sh", "-c"}
+
+		// fdisk is used to get all the partitions of the device.
+		// first all the partitions are cleared off any filesystem signatures, then the actual partition table
+		// header is removed. partprobe is called so as to re-read partition table, and update system with
```

Comment:
so partprobe is working?

Reply:
yes. if we give `partprobe /dev/sdX` its working. Earlier I tried using just `partprobe` command